docs/source/onnxruntime/usage_guides/models.mdx

```diff
@@ -185,6 +185,24 @@ prompt = "Face of a yellow cat, high resolution, sitting on a park bench"
 image = pipeline(prompt=prompt, image=init_image, mask_image=mask_image).images[0]
 ```
 
+### Panorama
+
+```python
+import torch
+from optimum.onnxruntime import ORTStableDiffusionPanoramaPipeline, ORTModelTiledVaeWrapper
+
+model_id = "runwayml/stable-diffusion-v1-5"
+pipeline = ORTStableDiffusionPanoramaPipeline.from_pretrained(model_id, revision="onnx")
+
+# patch pipeline with tiled VAE
+pipeline.vae_decoder = ORTModelTiledVaeWrapper(pipeline.vae_decoder, True, 64, 0.5)
+pipeline.vae_encoder = ORTModelTiledVaeWrapper(pipeline.vae_encoder, False, 64, 0.5)
```

Comment:
also I think we can integrate this in [`DiffusionPipelineMixin`](https://github.com/huggingface/optimum/blob/v1.13.2/optimum/pipelines/diffusers/pipeline_utils.py#L29) directly so that we can have instead : 

```suggestion
pipeline.enable_vae_tiling(64, 0.5)
```

Reply:
let me know if you need any support on that